app/models/user.rb

```diff
@@ -46,6 +46,16 @@ def last_week_tasks(date)
     project.tasks.where(updated_at: (date.beginning_of_week..date.end_of_week))
   end
 
+  def self.mentees_missing_on_website
+    mentees_missing = User.mentee.where("last_activity_at is null or last_activity_at >= (?)", 15.days.ago)
+    mentees_missing.where.not("is_missing = (?) OR (send_warning_email_after is not null AND send_warning_email_after > (?))", true, Date.today)
+  end
+
+  def self.mentors_missing_on_website
+    mentors_missing = User.mentor.where("last_activity_at is null or last_activity_at >= (?)", 15.days.ago)
+    mentors_missing.where.not("is_missing = (?) OR (send_warning_email_after is not null AND send_warning_email_after > (?))", true, Date.today)
```

Comment:
@anjali-sharma We forgot about one more thing - we also shouldn't send warning emails to people that were just matched. So we need to first check when the pair was created (sometimes we match existing users so checking just `last_activity_at` won't be enough).